### ISSUE TITLE
DNS TTL configurable

### ISSUE THREAD

**Issue by ThomasWaldmann** [nsupdate-info/nsupdate.info#51]
Samuirai wrote:
default is 60. create a user changeable field in a range.

**Comment by pejakm** [comment]
DnsExit service supports any value from 2 minutes to 48 hours.

**Comment by UnderEu** [comment]
https://blog.apnic.net/2019/11/12/stop-using-ridiculously-low-dns-ttls 

My personal hosts update IP addresses every 1 or 2 weeks, more or less; I have no need to fetch a new DNS query every 1 minute if my addresses won't change anytime sooner than that. I want the option to set the TTL I want on my hosts, even behind an Advanced panel or some kind of "I know what I'm doing" consent check. 

I'm no programmer, I can't contribute with code but I'm available to test things when necessary.


Oh... and this issue doesn't need to wait another 10+ years to be fixed.

**Comment by ThomasWaldmann** [comment]
I agree that for rarely changing "static" DNS entries one should of course not use very short TTLs.

But dyndns is usually used for dynamic IPs, the connections are less stable (and often even intentionally disconnected once a day) and often the IP changes when a new (e.g. DSL) connection is established. If one uses a long TTL, that means that after reconnecting, the IP in DNS will be **invalid** for that long time until all caching DNS servers have refreshed.

So, I personally won't work on this, but if someone wants to make a PR implementing this in the nsupdate.info software, I will review it.
